README.md

```diff
@@ -1,148 +1,28 @@
-# Istio Authentication
+# Istio Security
 
 [![Go Report Card](https://goreportcard.com/badge/github.com/istio/auth)](https://goreportcard.com/report/github.com/istio/auth)
 [![codecov](https://codecov.io/gh/istio/auth/branch/master/graph/badge.svg)](https://codecov.io/gh/istio/auth)
 
+Group alias: istio-security@googlegroups.com
 
-## Overview
+Istio Security SIG and sub workgroups:
 
-Istio Auth's aim is to enhance the security of microservices and their communication without requiring service code changes. It is responsible for:
+(1) Istio Security SIG
```

Comment:
Using bullets instead of (1), (2)... can give you better indention.

Reply:
i see, thanks, done.